plans/feat-marp-split-preview.md

```
@@ -0,0 +1,76 @@
+# feat(marp): side-by-side ライブプレビュー
+
+Issue: receptron/mulmoclaude#1647
+
+## 背景
+
+#1646 で chat plugin View の Marp 分岐に source editor (textarea) を露出した。これは「編集 → Apply → preview 更新」の離散ステップで、編集中はプレビューが古いまま。
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)